nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/PublicationLoaderServiceTest.java

```diff
@@ -61,12 +73,460 @@ void shouldGetExpectedDataFromExampleDocuments(String filename, PublicationDto e
     assertThat(actual).usingRecursiveComparison().ignoringCollectionOrder().isEqualTo(expected);
   }
 
+  @Test
```

Comment:
The tests in this file and the code they cover should be moved to validation of expanded resources